apps/condo/domains/billing/components/BillingPageContent/index.tsx

```
@@ -9,16 +9,20 @@ import { useOrganization } from '@open-condo/next/organization'
 import { Button, Space, Tag, Tooltip, Typography } from '@open-condo/ui'
 import { useBreakpoints } from '@open-condo/ui/hooks'
 
+import { useBillingAndAcquiringContexts } from '@condo/domains/billing/components/BillingPageContent/ContextProvider'
 import { useBillingHeaderTags } from '@condo/domains/billing/hooks/useBillingHeaderTags'
 import { AccessDeniedPage } from '@condo/domains/common/components/containers/AccessDeniedPage'
 import { PageWrapper, PageHeader } from '@condo/domains/common/components/containers/BaseLayout/BaseLayout'
 import { UI_BILLING_SPP_COMBINED_PAGE } from '@condo/domains/common/constants/featureflags'
+import {  CONTEXT_FINISHED_STATUS as BILLING_CONTEXT_FINISHED_STATUS } from '@condo/domains/billing/constants/constants'
```

ESLint

[error] `@condo/domains/billing/constants/constants` import should occur before import of `@condo/domains/billing/hooks/useBillingHeaderTags`

(import/order)